### ISSUE TITLE
require('react-easy-state') yields nothing.

### ISSUE THREAD

**Issue by fiatjaf** [solkimicreb/react-easy-state#2]
```js
fiatjaf@cantillon ~> node
> require('react-easy-state')
{}
> require('react-easy-state').default
undefined
```

**Comment by solkimicreb** [comment]
It is probably an issue with the ES5 webpack build, I will check it in a few hours.

Thx for the issue!

**Comment by solkimicreb** [comment]
Hi!

I released the next version (v1.0.5) which fixed the commonJS build issue and updated the readme.

Changelog is here: https://github.com/solkimicreb/react-easy-state/releases/tag/v1.0.5
Related readme section is here: https://github.com/solkimicreb/react-easy-state#with-commonjs-modules

Sorry for being so slow ):
